graylog2-web-interface/src/views/logic/searchtypes/messages/MessageSortConifg.js

```diff
@@ -0,0 +1,36 @@
+// @flow strict
+import Direction from 'views/logic/aggregationbuilder/Direction';
+import SortConfig, { Builder } from 'views/logic/aggregationbuilder/SortConfig';
+
+export type MessageSortConifgJson = {
```

Comment:
There is a typo in the class, file and type names: `MessageSortConifg` -> `MessageSortConfig`

Reply:
Good catch, thanks.